### PR TITLE
[WG][APB-8460]  Upgrade to play 3

### PR DESCRIPTION
**Pull request by gitwojciech**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,3 +1,23 @@
+import uk.gov.hmrc.{DefaultBuildSettings, SbtAutoBuildPlugin}
+
+
+val appName = "agent-subscription-frontend"
+
+ThisBuild / majorVersion := 1
```
Comment (r-melvin) [inline build.sbt:6]: This is also defined on line 31, should we increment this to 2 as well?

Comment (geoffreywatson) [reply build.sbt:6]: https://semver.org/ -- this is what I was going to review with the team

Comment (geoffreywatson) [reply build.sbt:6]: although we have incremented major version in agent-assurance going to Play 3.0 I believe we shouldn't have...?

Comment (r-melvin) [reply build.sbt:6]: I'm not overly precious on the versioning as long as it makes sense, you could argue that moving to play 3 is major change as we have to change the scala version, restructure the app, fix texts and be exposed to new bugs etc

**Pull request merged**